gcp/rakefiles/entrypoint.rake

```diff
@@ -87,28 +87,28 @@ task :set_current_project => [:set_vars, @gcp_creds_file, @serviceaccount_key_fi
 end
 
 desc "Create cluster and deploy GPII components to it"
-task :call_deploy => [:set_vars, @gcp_creds_file, @serviceaccount_key_file, @kubectl_creds_file] do
+task :deploy => [:set_vars, @gcp_creds_file, @serviceaccount_key_file, @kubectl_creds_file] do
   # Workaround for 'context deadline exceeded' issue:
   # https://github.com/exekube/exekube/issues/62
   # https://github.com/docker/for-mac/issues/2076
   # Remove this when docker for mac 18.05 becomes stable
   sh "docker run --rm --privileged alpine hwclock -s"
-  sh "#{@exekube_cmd} rake deploy"
+  sh "#{@exekube_cmd} rake xk[up]"
 end
 
 desc "Undeploy GPII compoments and destroy cluster"
-task :call_destroy => [:set_vars, @gcp_creds_file, @serviceaccount_key_file, @kubectl_creds_file] do
-  sh "#{@exekube_cmd} rake destroy"
+task :destroy => [:set_vars, @gcp_creds_file, @serviceaccount_key_file, @kubectl_creds_file] do
+  sh "#{@exekube_cmd} rake xk[down]"
 end
 
 desc "[ADVANCED] Create or update low-level infrastructure"
-task :call_apply_infra => [:set_vars, @gcp_creds_file, @serviceaccount_key_file] do
-  sh "#{@exekube_cmd} rake apply_infra"
+task :apply_infra => [:set_vars, @gcp_creds_file, @serviceaccount_key_file] do
+  sh "#{@exekube_cmd} up live/#{@env}/infra"
```

Comment:
Everything where secrets involved needs to go thru `rake xk`. Since there can be no secrets for infra by definition (we need infra to manage secrets), we can call `up live/#{@env}/infra` directly. I think it also looks better than `rake xk['up live/#{@env}/infra ',skip_infra,skip_secret_mgmt]`.